File: src/exercise04/tarefa/tarefa02/java/CamaroteInferior.java
package exercise04.tarefa.tarefa02.java;

public class CamaroteInferior extends VIP {
    private String location;

    public CamaroteInferior(double value, double additionalValue, String location) {
        super(value, additionalValue);
        this.location = location;
    }

    @Override
    public double imprimeValor() {
        double finalPrice = getValue() + getAdditionalValueVIP();
        System.out.println("Ingresso VIP Camarote Inferior = R$ " + finalPrice);
        return finalPrice;
    }

    public void imprimeLocalizacao() {
        System.out.println("Localização: " + location);
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }


}

File: src/exercise04/tarefa/tarefa02/kotlin/CamaroteInferior.kt
package exercise04.tarefa.tarefa02.kotlin

class CamaroteInferior(
    value: Double,
    additionalValue: Double,
    var location: String
) :
    VIP(value, additionalValue) {

    override fun imprimeValor(): Double {
        val finalPrice = value + additionalValueVIP
        println("Ingresso VIP Camarote Inferior = R$ $finalPrice")
        return finalPrice
    }

    fun imprimeLocalizacao() {
        println("Localização: $location")
    }
}